src/App.jsx

```diff
@@ -1,10 +1,98 @@
+import { useEffect, useState } from "react";
+import "react-mde/lib/styles/css/react-mde-all.css";
+import Sidebar from "./components/Sidebar";
+import Editor from "./components/Editor";
+import Split from "react-split";
+import { nanoid } from "nanoid";
+
+
 export default function App() {
+
+  const [notes, setNotes] = useState(() => JSON.parse(localStorage.getItem('notes')) || [])
+
+
+  const [currentNoteId, setCurrentNoteId] = useState(
+    (notes[0] && notes[0].id) || ""
```

Comment:
To improve your code readability you can make this a ternary operator 
```suggestion
    notes[0] ? notes[0].id : ""
```